GitCommands/Config/ConfigFile.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace GitCommands.Config
 {
-    public class ConfigFile
+    public class ConfigFile : ISubmodulesConfigFile
```

Comment:
❓ I read this as "Every config file is a submodules config file". This feels wrong.
The interface `ISubmodulesConfigFile` does not contain anything submodule-specific at a first glance.

Reply:
It is the data in .gitmodules
The interface needs some comments...